ql/src/test/queries/clientpositive/ctas_uses_database_location.q

```diff
@@ -1,5 +1,5 @@
 --! qt:dataset:src
-set hive.metastore.warehouse.dir=invalid_scheme://${system:test.tmp.dir};
+-- set hive.metastore.warehouse.dir=invalid_scheme://${system:test.tmp.dir};
```

Comment:
In a previous patch it had to be commented because we were defaulting to `hive.metastore.warehouse.dir` if the external warehouse directory so this doesn't have to be commented any more. I'll remove the comment.